Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -178,11 +178,14 @@ extension File {
         return matches.filter { !$0.intersectsRanges(exclusionRanges) }
     }
 
-    internal func validateVariableName(_ dictionary: [String: SourceKitRepresentable],
-                                       kind: SwiftDeclarationKind) -> (name: String, offset: Int)? {
+    internal func validateIdentifierName(_ dictionary: [String: SourceKitRepresentable],
+                                     kind: SwiftDeclarationKind) -> (name: String, offset: Int)? {
```

Comment:
that was probably intentional to avoid breaking the return 😅 
either way, I've moved this to the rule itself, since IMO it's not related to `File` in any obvious way.